### ISSUE TITLE
Connecting to Tilt Cloud via Tilt UI is not accessible via Voiceover

### ISSUE THREAD

**Issue by hyu** [tilt-dev/tilt#4424]
I received a report via email:

> Connecting to the Tilt Cloud via the Tilt UI may not be accessible. I’ve tried with both Chrome and Safari, and VoiceOver recognizes that a dialog has opened (either when I click “Make Snapshot” or “Account” > “Connect Tilt To Tilt Cloud”), but I can’t seem to do any kind of interaction with the dialog at all.

We should check to make sure that the buttons to create the snapshot can be activated via Voiceover.
